langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaDateDeserializer.java

```diff
@@ -0,0 +1,32 @@
+package dev.langchain4j.model.ollama;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+
+import java.io.IOException;
+import java.time.OffsetDateTime;
+import java.time.format.DateTimeFormatter;
+
+class OllamaDateDeserializer extends JsonDeserializer<OffsetDateTime> {
+
+    // Ollama returns different lengths of the nanosecond part in the date; for some models, it is 8 digits, for others 9.
+    // This method removes the nanoseconds, as this level of precision isn't important.
+    // input      -> 2024-08-04T00:54:54.764563036+02:00
+    // output     -> 2024-08-04T00:54:54.+02:00
+    @Override
+    public OffsetDateTime deserialize(JsonParser p, DeserializationContext ctxt) throws IOException {
```

Comment:
could you pelase write a unit test for this?

Reply:
I've added this with a new commit. Please check if it is sufficient.